cpp/tests/test_graph_simulation.cpp

```diff
@@ -121,6 +125,32 @@ TEST(TestGraphSimulation, stopsAtTmax)
     EXPECT_NEAR(sim.get_t(), tmax, 1e-15);
 }
 
```

Comment:
Since the algorithm we test is stochastic, the results depend on the RNG (Random Number Generator), which generates numbers depending on some seed. The seed is usually chosen at random, but if we fix the seed (i.e. "seeding" the RNG), the generated random numbers (and thus the simulation result) will be the same every time the code is run.

Testing the result of a simulation would ensure that the simulation is not (meaningfully) influenced by some future change to another component (e.g. the RNG).